### ISSUE TITLE
scipy module fails loading for 32 bit and 64 bit APK builds. 

### ISSUE THREAD

**Issue by brentpicasso** [kivy/python-for-android#2061]
### Versions

* Python: 3.7.1
* OS: Ubuntu 18
* Kivy: 1.11.1
* Cython: 0.29

### Description
Including scipy yields an error when loading the module. Different errors for 32 bit and 64 bit pacakges. 

Error for 32 bit packages:

dlopen failed: "/data/data/com.autosportlabs.podium/files/app/_python_bundle/site-packages/scipy/_lib/_ccallback_c.so" is 64-bit instead of 32-bit

Error for 64 bit pacakges:

dlopen failed: "/data/data/com.autosportlabs.podium/files/app/_python_bundle/site-packages/scipy/_lib/_ccallback_c.so" has unexpected e_machine: 62

### buildozer.spec
```
[app]

# (str) Title of your application
title = Podium

# (str) Package name
package.name = podium

# (str) Package domain (needed for android/ios packaging)
package.domain = com.autosportlabs

# (str) Source code where the main.py live
source.dir = .

# (list) Source files to include (let empty to include all the files)
source.include_exts = py,png,jpg,kv,atlas,ttf,json,svg,cfg,obj,mtl,glsl,zip,sql

# (list) Source files to exclude (let empty to not exclude anything)
#source.exclude_exts = spec

# (list) List of directory to exclude (let empty to not exclude anything)
source.exclude_dirs = tests, bin, .git, install, build_tools

# (list) List of exclusions using pattern matching
#source.exclude_patterns = license,images/*/*.jpg

# (str) Application versioning (method 1)
version.regex = __version__ = ['"](.*)['"]
version.filename = %(source.dir)s/main.py

# (str) Application versioning (method 2)
# version = 1.2.0

# (list) Application requirements
# comma seperated e.g. requirements = sqlite3, kivy, pygments

requirements = python3==3.7.1, kivy==1.11.1, numpy, scipy, android, websocket-client, raven==6.10.0, pyjnius, plyer, podium_api==0.0.14, https://github.com/nskrypnik/kivy3/zipball/master
# ffpyplayer==v4.1.0, ffpyplayer_codecs, 

# (list) Garden requirements
#garden_requirements =

# (str) Presplash of the application
presplash.filename = resource/images/app_splash_screen.png

# (str) Icon of the application
icon.filename = resource/images/app_icon_256x256.png

# (str) Supported orientation (one of landscape, portrait or all)
orientation = all

# (bool) Indicate if the application should be fullscreen or not
fullscreen = 1


#
# Android specific
#

# (list) Permissions
android.permissions = INTERNET, BLUETOOTH, BLUETOOTH_ADMIN , WRITE_EXTERNAL_STORAGE, WAKE_LOCK, ACCESS_FINE_LOCATION, CHANGE_WIFI_STATE, CHANGE_NETWORK_STATE, ACCESS_WIFI_STATE, ACCESS_NETWORK_STATE

# (int) Android API to use
android.api = 28

# (int) Minimum API required (8 = Android 2.2 devices)
android.minapi = 21

# (str) Android NDK version to use
android.ndk = 19b

# (int) Android NDK API to use (optional)
android.ndk_api = 21

# (bool) Use --private data storage (True) or --dir public storage (False)
#android.private_storage = True

# (str) Android NDK directory (if empty, it will be automatically downloaded.)
#android.ndk_path =

# (str) Android SDK directory (if empty, it will be automatically downloaded.)
#android.sdk_path =

p4a.branch = release-2019.10.06

# (str) python-for-android git clone directory (if empty, it will be automatically cloned from github)
#android.p4a_dir = ../python-for-android

# (list) python-for-android whitelist
#android.p4a_whitelist =

# (str) Android entry point, default is ok for Kivy-based app
#android.entrypoint = org.renpy.android.PythonActivity

# (list) List of Java .jar files to add to the libs so that pyjnius can access
# their classes. Don't add jars that you do not need, since extra jars can slow
# down the build process. Allows wildcards matching, for example:
# OUYA-ODK/libs/*.jar
#android.add_jars = jar/*.jar
#android.add_jars = foo.jar,bar.jar,path/to/more/*.jar
android.add_src = java/src/

# (list) List of Java files to add to the android project (can be java or a
# directory containing the files)
#android.add_src =
#android.add_src = java/src/

# (str) python-for-android branch to use, if not master, useful to try
# not yet merged features.
#android.branch = master

# (str) OUYA Console category. Should be one of GAME or APP
# If you leave this blank, OUYA support will not be enabled
#android.ouya.category = GAME

# (str) Filename of OUYA Console icon. It must be a 732x412 png image.
#android.ouya.icon.filename = %(source.dir)s/data/ouya_icon.png

# (str) XML file to include as an intent filters in <activity> tag
#android.manifest.intent_filters =

# (list) Android additionnal libraries to copy into libs/armeabi
#android.add_libs_armeabi = libs/android/*.so
#android.add_libs_armeabi_v7a = libs/android-v7/*.so
#android.add_libs_x86 = libs/android-x86/*.so
#android.add_libs_mips = libs/android-mips/*.so

# (bool) Indicate whether the screen should stay on
# Don't forget to add the WAKE_LOCK permission if you set this to True
android.wakelock = True

# (list) Android application meta-data to set (key=value format)
#android.meta_data =

# (list) Android library project to add (will be added in the
# project.properties automatically.)
#android.library_references =

#
# iOS specific
#

# (str) Name of the certificate to use for signing the debug version
# Get a list of available identities: buildozer ios list_identities
#ios.codesign.debug = "iPhone Developer: <lastname> <firstname> (<hexstring>)"

# (str) Name of the certificate to use for signing the release version
#ios.codesign.release = %(ios.codesign.debug)s


[buildozer]

# (int) Log level (0 = error only, 1 = info, 2 = debug (with command output))
log_level = 2

# (int) Display warning if buildozer is run as root (0 = False, 1 = True)
warn_on_root = 1


# -----------------------------------------------------------------------------
# List as sections
#
# You can define all the "list" as [section:key].
# Each line will be considered as a option to the list.
# Let's take [app] / source.exclude_patterns.
# Instead of doing:
#
#     [app]
#     source.exclude_patterns = license,data/audio/*.wav,data/images/original/*
#
# This can be translated into:
#
#     [app:source.exclude_patterns]
#     license
#     data/audio/*.wav
#     data/images/original/*
#


# -----------------------------------------------------------------------------
# Profiles
#
# You can extend section / key with a profile
# For example, you want to deploy a demo version of your application without
# HD content. You could first change the title to add "(demo)" in the name
# and extend the excluded directories to remove the HD content.
#
#     [app@demo]
#     title = My Application (demo)
#
#     [app:source.exclude_patterns@demo]
#     images/hd/*
#
# Then, invoke the command line with the "demo" profile:
#
#     buildozer --profile demo android debug

```

Command:

32 bit builds:
```
export APP_ANDROID_ARCH=armeabi-v7a
buildozer android debug
```
64 bit builds:
```
export APP_ANDROID_ARCH=arm64-v8a
buildozer android debug
```

**Comment by Ham-Merhead** [comment]
scipy is not pure Python and does not have a recipe - this is not going to work.

The messages are due to pip downloading the binaries for some other architecture, probably because the wheel is named without considering Android.

**Comment by misl6** [comment]
Fixed via #2370
